### PR TITLE
Hosted CE documentation migrated from helpdesk (SOFTWARE-3445)

### PR DESCRIPTION
**Pull request by matyasselmeci**
Original page: https://support.opensciencegrid.org/support/solutions/articles/12000025149-osg-managed-services

### CODE REVIEW

**Comment by matyasselmeci** [comment]
This doesn't have information on the WN client yet, but I'd like to get a preliminary review.

File: docs/compute-element/hosted-ce.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+OSG Hosted CE
+=============
+
+
+## Overview
```
Comment (brianhlin) [inline docs/compute-element/hosted-ce.md]: - https://opensciencegrid.org/technology/documentation/style-guide/#headings
- https://opensciencegrid.org/technology/documentation/writing-documentation/#introductions

File: docs/compute-element/hosted-ce.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+As an alternative to setting up your own CE, OSG offers a hosted CE option
+to connect your existing campus HPC/HTC cluster to the Open Science Grid.
+The OSG team will host and operate the HTCondor compute element which routes user jobs to your cluster,
+configured for the science communities that you choose to support.
+
+![managed services diagram](/compute-element/img/screenshot_2983.png)
+
+
+## Requirements
```
Comment (brianhlin) [inline docs/compute-element/hosted-ce.md]:
```suggestion
## Before Starting
```

File: docs/compute-element/hosted-ce.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+Here are the requirements:
```
Comment (brianhlin) [inline docs/compute-element/hosted-ce.md]:
```suggestion
Before preparing your cluster for OSG jobs, consider the following points:
```

File: docs/compute-element/hosted-ce.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+-   An existing compute cluster with a supported batch system (HTCondor, LSF, PBS, Slurm, SGE),
+    and a supported operating system (RHEL 6 or 7 or compatible)
```
Comment (brianhlin) [inline docs/compute-element/hosted-ce.md]: > -   An existing compute cluster with a [supported batch system](/index#prepare-the-batch-system) running on a
    [supported operating system](release/supported_platforms)

File: docs/compute-element/hosted-ce.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+-   Outbound network connectivity from the compute nodes (they can be behind NAT)
+
+-   A Unix account on your cluster's submit server, accessible via an SSH key
+    The OSG CE will use this account to automatically submit jobs,
+    so it must also have permissions to submit jobs to the batch system
+
+-   If your batch system is not HTCondor,
+    there needs to be a shared file system between the submit server and the compute nodes
```
Comment (brianhlin) [inline docs/compute-element/hosted-ce.md]:
```suggestion
    there must be a shared file system between the submit server and the compute nodes
```

File: docs/compute-element/hosted-ce.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+
+## Overview
+
+As an alternative to setting up your own CE, OSG offers a hosted CE option
+to connect your existing campus HPC/HTC cluster to the Open Science Grid.
+The OSG team will host and operate the HTCondor compute element which routes user jobs to your cluster,
+configured for the science communities that you choose to support.
+
+![managed services diagram](/compute-element/img/screenshot_2983.png)
```
Comment (brianhlin) [inline docs/compute-element/hosted-ce.md]: We keep images in `docs/img/` but I think I like this better. However, can we get more descriptive names for these two images?

File: docs/compute-element/hosted-ce.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+in order to support application software repositories.
+This will allow a broader set of jobs to run on your cluster.
+
+
+## Security
+
+OSG takes multiple precautions to maintain security and prevent unauthorized
+usage of resources:
+
+-   Access to the OSG system with ssh keys are restricted to the OSG staff maintaining them
```
Comment (brianhlin) [inline docs/compute-element/hosted-ce.md]:
```suggestion
-   Access to the OSG system with SSH keys are restricted to the OSG staff maintaining them
```

File: docs/compute-element/hosted-ce.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+-   Users are carefully vetted before they are allowed to submit jobs to OSG
+-   Jobs running through OSG can be traced back to the user that submitted them
+-   Job submission can quickly be disabled if needed
+-   OSG staff are readily contactable in case of an emergency,
+    through email at [help@opensciencegrid.org](help@opensciencegrid.org)
```
Comment (brianhlin) [inline docs/compute-element/hosted-ce.md]:
```suggestion
    through email at <mailto:help@opensciencegrid.org>
```

File: docs/compute-element/hosted-ce.md [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+-   Validate operation with a set of test jobs
+
+-   Configure central OSG services to schedule jobs
+
+Optionally, we can assist you in installing and setting up the Squid and OASIS software on your cluster
+in order to support application software repositories.
+This will allow a broader set of jobs to run on your cluster.
+
+
+## Security
```
Comment (brianhlin) [inline docs/compute-element/hosted-ce.md]: Let's move the security section right below the requirements/before starting section

File: docs/compute-element/hosted-ce.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+
+-   A Unix account on your cluster's submit server, accessible via an SSH key
+    The OSG CE will use this account to automatically submit jobs,
+    so it must also have permissions to submit jobs to the batch system
+
+-   If your batch system is not HTCondor,
+    there needs to be a shared file system between the submit server and the compute nodes
+
+
+## Setup Process
```
Comment (brianhlin) [inline docs/compute-element/hosted-ce.md]:
```suggestion
## Applying for an OSG Hosted CE
```

File: docs/compute-element/hosted-ce.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+
+-   Access to the OSG system with ssh keys are restricted to the OSG staff maintaining them
+-   Users are carefully vetted before they are allowed to submit jobs to OSG
+-   Jobs running through OSG can be traced back to the user that submitted them
+-   Job submission can quickly be disabled if needed
+-   OSG staff are readily contactable in case of an emergency,
+    through email at [help@opensciencegrid.org](help@opensciencegrid.org)
+
+
+## Which Science Communities and Institutions am I Supporting?
```
Comment (brianhlin) [inline docs/compute-element/hosted-ce.md]: Let's move this section above security and below the requirements/before starting section

File: docs/compute-element/hosted-ce.md (hunk 6 above)
Comment (matyasselmeci) [reply docs/compute-element/hosted-ce.md]: You mean for the file path?

File: docs/compute-element/hosted-ce.md (hunk 8 above)
Comment (matyasselmeci) [reply docs/compute-element/hosted-ce.md]: Does the `mailto:` show up in the text?

File: docs/compute-element/hosted-ce.md (hunk 6 above)
Comment (brianhlin) [reply docs/compute-element/hosted-ce.md]: Yup, sorry for the vagueness

File: docs/compute-element/hosted-ce.md (hunk 3 above)
Comment (matyasselmeci) [reply docs/compute-element/hosted-ce.md]: "consider the following points" does not seem strong enough. This is stuff you gotta do.

File: docs/compute-element/hosted-ce.md [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+configured for the science communities that you choose to support.
+
+![managed services diagram](/compute-element/img/screenshot_2983.png)
+
+
+## Requirements
+
+Here are the requirements:
+
+-   An existing compute cluster with a supported batch system (HTCondor, LSF, PBS, Slurm, SGE),
```
Comment (matyasselmeci) [inline docs/compute-element/hosted-ce.md]:
```suggestion
-   An existing compute cluster with a [supported batch system](/index#prepare-the-batch-system)
```

File: docs/compute-element/hosted-ce.md (hunk 4 above)
Comment (matyasselmeci) [inline docs/compute-element/hosted-ce.md]:
```suggestion
    running on a [supported operating system](release/supported_platforms)
```

File: docs/compute-element/hosted-ce.md (hunk 3 above)
Comment (brianhlin) [reply docs/compute-element/hosted-ce.md]: That's a fair point, I just wanted to expand on the context more and copy/pasted what we had in the CE installation guide. How about this?

> Before preparing your cluster for OSG jobs, consider the following requirements:

Comment (matyasselmeci) [reply docs/compute-element/hosted-ce.md]: Sounds fine. Can you update the suggestion?

File: docs/compute-element/hosted-ce.md (hunk 8 above)
Comment (brianhlin) [reply docs/compute-element/hosted-ce.md]: Nope. See it in markdown [here](https://github.com/opensciencegrid/docs/blame/master/docs/site-planning.md#L59) and rendered [here](https://opensciencegrid.org/docs/site-planning/#pilot-jobs)

**Pull request merged**